src/style.css

```diff
@@ -0,0 +1,58 @@
+html,
+body,
+section,
+div {
+  margin: 0;
+  padding: 0;
+}
+
+.stars-container {
+  margin: 0 auto;
+  width: 96px;
+  height: 96px;
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  height: 16px;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
```

Comment:
Doublecheck stars size and indent between stars from Figma 
![image](https://github.com/mate-academy/layout_stars/assets/80700247/bd68bb30-bc94-418c-922b-dc48479b6d1e)